src/components/PerfherderGraph/index.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
+import HelpOutlineIcon from '@material-ui/icons/HelpOutline';
```

Comment:
I think the icon describes what we need here. 

It may have been better to have an [icon like `InfoOutline`](https://www.materialui.co/icon/info-outline), but unfortunately there seems to be only a filled info that does not look that good when the others are outlined:

![inco-info](https://user-images.githubusercontent.com/51707553/111332421-6f843700-867a-11eb-9504-967f01129104.PNG)